src/game.zig

```diff
@@ -567,6 +567,9 @@ pub const Player = struct { // MARK: Player
 
 	pub fn placeBlock() void {
 		if(main.renderer.MeshSelection.selectedBlockPos) |blockPos| {
+			if(!main.KeyBoard.key("shift").pressed)
+				if(main.renderer.mesh_storage.triggerOnInteractBlock(blockPos[0], blockPos[1], blockPos[2]) == .handled) return;
+
```

Comment:
I don't like how this is formatted. I would prefer explicitly adding braces around the outer if here.
```suggestion
			if(!main.KeyBoard.key("shift").pressed) {
				if(main.renderer.mesh_storage.triggerOnInteractBlock(blockPos[0], blockPos[1], blockPos[2]) == .handled) return;
			}
```